MonoGame.Framework/MonoGame.Framework.ConsoleCheck.csproj

```diff
@@ -6,7 +6,7 @@
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <UseWindowsForms>true</UseWindowsForms>
     <Description>Fake console assembly that checks if MonoGame's code is still compatible with the private console implementations and building against .NET 4.5.2 and C# 5.</Description>
-    <LangVersion>5</LangVersion>
+    <LangVersion>11</LangVersion>
```

Comment:
@harry-cpp and i discussed this.  It seems to work fine for just language features on BRUTE, NativeAOT, and even old Mono console runtimes.